src/control/jobs/CustomExportJob.cpp

```diff
@@ -39,6 +39,11 @@ CustomExportJob::~CustomExportJob()
 	}
 	exportRange.clear();
 
+	for(auto& filter : filters)
```

Comment:
If you add the class by Value instead of pointer, you don't have to delete it. Even if this is slower, usually it doesen't matter, and you don't have to care about.
This is not a Problem, only a Hint ;-)

Reply:
Yes but I thought that since in the Contructor I create a `new ExportType`, I'd have to delete it in deconstruction phase, isn't it?

Reply:
Yes, but if you dont write new, and remove the start in the type, it will be automatically be deleted.
Each time you access it (without reference) it will be copied, which is inefficent, but in such small cases it will be OK and easier to implement. It don't make sense, to change this now, as this is technical the better solution, but it's simpler the other way ;-)